mandatory/3-function-output.js

```diff
@@ -1,14 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
-}
+} // The Math.random() function returns a random number that is greater that or equal to 0 and less than 1 which you can then scale to your desired range.
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+// The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.
+
 function concatenate(firstWord, secondWord, thirdWord) {
+  const cancWords = firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
Do you think const is necessary here?
```suggestion
  const cancWords = firstWord.concat(" ", secondWord, " ", thirdWord);
```
return firstWord.concat(" ", secondWord, " ", thirdWord);